help/destinations/catalog/crm/salesforce.md

```diff
@@ -121,26 +135,27 @@ Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]** search for [!DNL
 
 ### Authenticate to destination {#authenticate}
 
-To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+To authenticate to the destination, fill in the required fields below and select **[!UICONTROL Connect to destination]**. Refer to the [Gather [!DNL Salesforce CRM] credentials](#gather-credentials) section for any guidance.
+*  **[!UICONTROL Username]**: Your Salesforce account username.
+*  **[!UICONTROL Password]**: Your Salesforce account password.
+*  **[!UICONTROL Custom Domain]**: Your Salesforce domain prefix. For example if your domain is <br> <i>`d5i000000isb4eak-dev-ed`.my.salesforce.com</i>, you need the highlighted value.
+*  **[!UICONTROL Client ID]**: Your Salesforce connected app `Consumer Key`. Refer to the [Salesforce documentation](https://help.salesforce.com/s/articleView?id=sf.connected_app_rotate_consumer_details.htm&type=5) if you need additional guidance.
+*  **[!UICONTROL Client Secret]**: Your Salesforce connected app `Consumer Secret`. Refer to the [Salesforce documentation](https://help.salesforce.com/s/articleView?id=sf.connected_app_rotate_consumer_details.htm&type=5) if you need additional guidance.
 
 ![Platform UI screenshot showing how to authenticate.](../../assets/catalog/crm/salesforce/authenticate-destination.png)
 
-*  **[!UICONTROL Password]**: Your Salesforce account password.
-*  **[!UICONTROL Custom Domain]**: Your Salesforce domain.
-*  **[!UICONTROL Client ID]**: Your Salesforce connected app Consumer Key.
-*  **[!UICONTROL Client Secret]**: Your Salesforce connected app Consumer Secret.
-*  **[!UICONTROL Username]**: Your Salesforce account username.
-
 If the details provided are valid, the UI displays a **[!UICONTROL Connected]** status with a green check mark, you can then proceed to the next step.
 
 ### Fill in destination details {#destination-details}
 
 To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
-![Platform UI screenshot showing the destination details.](../../assets/catalog/crm/salesforce/destination-details.png)
-
 *  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
-*  **[!UICONTROL Salesforce ID Type]**: Select **[!UICONTROL Contact]** if the identities you are looking to export or update are of type *Contact*. Select **[!UICONTROL Lead]** if the identities you are looking to export or update are of type *Lead*.
+*  **[!UICONTROL Salesforce ID Type]**: 
+    * Select **[!UICONTROL Contact]** if the identities you are looking to export or update are of type *Contact*.
```

Comment:
Question here: How would I know if I am exporting Contacts or Leads? Are there any best practices around when identities should be contacts or leads? Just a hunch - are Leads used more in a B2B (business-to-business) context, since there is an additional required field `Company` for them?

Reply:
Truth be said there is no technical distinction between a lead and a contact both are the same. The distinction however has already been added to the overview section. 

```
* [Leads](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_lead.htm) - A lead is the name of a person or company who may (or may not) be interested in the products or services you sell.
* [Contacts](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_contact.htm) - A contact is an individual with whom one of your representatives has established a relationship and has been qualified as a potential customer.   
```